scripts/ipfs/install

```diff
@@ -29,9 +29,25 @@ ipfs init || true
 # Enable gossipsub routing
 ipfs config Pubsub.Router gossipsub
 
+# Enable relay hop to help peers with connecting
+ipfs config --bool Swarm.EnableRelayHop true
+
 # Enable Filestore for --nocopy capability
 ipfs config --bool Experimental.FilestoreEnabled true
 
+# Disable NAT port mapping used for NAT traversal
+ipfs config --bool DisableNatPortMap true
```

Comment:
What about pis connected to the home internet, your killing their ability to try to gain more peer by nat port.

Reply:
Yeah, I mentioned that in my description but didn't change it. I will change it back.